cypress/integration/frontsite/literatureDetails.test.spec.js

```diff
@@ -46,21 +52,24 @@ describe('frontsite literature details', () => {
           is_requested: isRequested,
         } = xhr.response.body;
         expect(hasActiveLoan).to.be.false;
-        if (isRequested) {
+
+        if (!isRequested) {
           cy.contains('Have it delivered to my office').click();
 
           cy.get('input[name=selectedDate]').click();
 
           cy.wait('@location').wait(1000); // Cannot avoid this delay
-          cy.get('td:not(.disabled) > span.suicr-content-item')
+          const dayFromDatePicker =
+            'td:not(.disabled) > span.suicr-content-item';
+          cy.get(dayFromDatePicker)
             .first()
             .type('{enter}');
 
           cy.contains('button.primary', 'Request').click();
-          cy.contains('div.positive.message', 'Request created');
 
-          cy.wait(1000); // ES delay
+          cy.wait(5000); // ES delay
```

Comment:
```suggestion
          cy.wait(2000); // ES delay
```
It is true that `1` sec might not be enough, but `2` should be ok :)